tests/test_runner/plugin_launcher.py

```diff
@@ -27,6 +28,7 @@
                     "Runs plugins launcher and check if it runned properly"),
 )
 
+crun_input = '{"pid":12345}'
```

Comment:
We know from the OCI spec that crun will provide the container state over stdin in:

> The state of the container MUST be passed to hooks over stdin so that they may do work appropriate to the current state of the container.

and the state is defined as

> When serialized in JSON, the format MUST adhere to the following pattern:
> ```
> {
>    "ociVersion": "1.0.2",
>    "id": "oci-container1",
>    "status": "running",
>    "pid": 4422,
>    "bundle": "/containers/redis",
>    "annotations": {
>        "myKey": "myValue"
>    }
> }
> ```

Would probably be worth using this whole json block rather than just the PID for a more accurate test

Reply:
Added full JSON instead short one.